platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/state/service/WritableRosterStore.java

```diff
@@ -39,6 +38,8 @@
  * Read-write implementation for accessing rosters states.
  */
 public class WritableRosterStore extends ReadableRosterStoreImpl {
+    public static final String ROSTER_KEY = "ROSTERS";
+    public static final String ROSTER_STATES_KEY = "ROSTER_STATE";
```

Comment:
As mentioned above, we put these constants into the `RosterStateId` to be able to sue them in other places. Please note that the Readable store also uses them. It's not a good idea to duplicate these string literals. I suggest to keep using the `RosterStateId`.

Reply:
As above.